src/api/flaskr/service/shifu/route.py

```
@@ -1427,6 +1460,48 @@ def admin_operation_user_credit_grant(user_bid: str):
+                    schema:
+                        $ref: "#/components/schemas/AdminOperationUserPackageGrantRequestDTO"
+        responses:
+            200:
+                description: Operator user package grant result
+        """
+        _require_operator()
+        try:
+            payload = AdminOperationUserPackageGrantRequestDTO.model_validate(
+                request.get_json() or {}
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)